src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/Arbeidsgiverperiode.kt

```diff
@@ -9,6 +9,7 @@ import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.utils.valider
 @Serializable
 data class Arbeidsgiverperiode(
     val perioder: List<Periode>,
+    // TODO vurder å fjerne om man heller kan utlede fra AGP og sykmeldingsperioder
```

Comment:
Lar denne stå til den er mer avklart.

Reply:
Egenmeldingerfeltet kan sies å være overflødig fordi vi kan finne ut hvilke dager som er egenmeldinger ved å se på AGP og sykmeldingene. Dersom vi ønsker det så kan vi da fjerne `egenmeldinger`-feltet. Det er dog ikke helt rett frem mtp. hvordan frontenden fungerer og det kan bli forvirrende for bruker.